locations/spiders/barlouie.py

```diff
@@ -0,0 +1,48 @@
+import json
+
+import scrapy
+
+from locations.items import GeojsonPointItem
+
+
+class BarlouieSpider(scrapy.Spider):
+    name = "barlouie"
+    allowed_domains = ["www.barlouie.com"]
+    start_urls = ["https://www.barlouie.com/locations"]
+    item_attributes = {"brand": "Barlouie", "brand_wikidata": "Q16935493"}
```

Comment:
Used `https://www.wikidata.org/` for finding the brand_wikidata code. Please check once

Reply:
@Cj-Malone let me know if this is the right way to look for the brand_wikidata code. I can update the template in the readme file with this. (ex. `    item_attributes = {"brand": "Barlouie", "brand_wikidata": "Q16935493"}`) and can also add how to search for brand_wikidata code.

Reply:
Yeah, you can sometime find it in NSI too, either via [NSI](nsi.guide) or `pipenv run scrapy nsi --name "KFC"`